src/main/java/com/minecolonies/coremod/network/messages/server/BuildToolPlaceMessage.java

```diff
@@ -380,7 +380,6 @@ private static void setupBuilding(
             if (!(building instanceof IRSComponent))
             {
                 ConstructionTapeHelper.removeConstructionTape(building.getCorners(), world);
-                building.calculateCorners();
```

Comment:
If the corner calculation is not needed, the removeConstructionTape shouldn't be either. Needs some good testing though.

Reply:
corner calc happens before now, inbuilt

Reply:
so the removal can go away.

Reply:
but isnt that to remove construction tape? Shouldnt have anything todo with the corner calc

Reply:
Yeah I'm not 100% sure tbh. I'm not sure anymore why we removed and re-add here and especially why we did the corner calc inbetween.

Reply:
The remove and re-add is likely _because_ the calc was between.  In case it changed, it needs to move the tape.  So yeah, if you're not recalculating there any more it probably doesn't need to do either thing with the tape.  Unless the tape might not be there at all at this point?

Reply:
I guess from the looks of it it was an old hacky fix for removing previously set tape at a different location than normal building corners, possibly from times where we didnt load the schematic to check its size yet. I'll just remove it and we'll see what happens xD